### PR TITLE
Chore: dependency updates

### PR DESCRIPTION
**Pull request by mpetrunic**
- updates eslint to latest version
   - removes warning about unsupported typescript version
   - "ban-ts-ignore" rule put to warning - we are using it pretty lightly as there are bunch of places where it shouldn't be used
- removed babel-runtime dependency and plugin
   - it it automatically added to support old browsers without Promises and esmodules but dropped those with bigint anyways (most bundlers will polyfill this stuff anyways)

### CODE REVIEW

**Comment by mpetrunic** [comment]
I need #574 before we can merge because already released bls depends on babel-runtime

File: .babelrc [hunk 1]
```diff
@@ -2,14 +2,21 @@
   "presets": [
     [
       "@babel/env",
-      {"targets": {"node":"current"}}
+      {
+        "targets": {
+          "node": "8",
```
Comment (wemeetagain) [inline .babelrc]: I think we'll be using features only in current LTS and beyond (eg: bigint)

**Review by wemeetagain — DISMISSED** [review]

File: .babelrc (hunk 1 above)
Comment (mpetrunic) [reply .babelrc]: oh yeah, >=10.4.0 supports bigint

Although, I would still keep node 8 support for libraries without bigint. What do you think?

Comment (wemeetagain) [reply .babelrc]: :+1: Yeah, as much support as possible

File: .babelrc [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
+          "browsers": ">1%, not ie 11"
```
Comment (ec2) [inline .babelrc:8]: :( no love for IE

Comment (mpetrunic) [reply .babelrc:8]: haha, just noticed that bigint also means we dropped support for safari haha

Comment (ec2) [reply .babelrc:8]: rip. thats actually quite a big chunk of people

**Review by ec2 — DISMISSED** [review]

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**